dao/src/main/java/org/thingsboard/server/dao/sqlts/AbstractChunkedAggregationTimeseriesDao.java

```diff
@@ -119,7 +119,7 @@ public ListenableFuture<List<ReadTsKvQueryResult>> findAllAsync(TenantId tenantI
     @Override
     public ListenableFuture<ReadTsKvQueryResult> findAllAsync(TenantId tenantId, EntityId entityId, ReadTsKvQuery query) {
         var aggParams = query.getAggParameters();
-        if (Aggregation.NONE.equals(aggParams.getAggregation())) {
+        if (Aggregation.NONE.equals(aggParams.getAggregation()) || aggParams.getInterval() < 1) {
```

Comment:
Why did you leave this check if it won't pass the validation?

Reply:
maybe it's paranoia, but I'd leave it just in case